go/database/mpt/nibble.go

```diff
@@ -64,6 +64,24 @@ func KeyToNibblePath(key common.Key, source NodeSource) []Nibble {
 	return res
 }
 
+// addressToHashedPath converts the given path into a slice of Nibbles.
```

Comment:
It might be reasonable to add a few small unit tests for those helper functions.

Reply:
added. Just I created basically a new function in tests that hashes and converts to nibble. I do not know how to test it better